packages/edit-site/src/components/page/style.scss

```diff
@@ -2,16 +2,20 @@
 	color: $gray-800;
 	background: $white;
 	height: 100%;
+	container: edit-site-page / inline-size; /* stylelint-disable */
```

Comment:
The container queries are in the dataviews package and the container is outside, this is probably the only thing that I'd change, I'd move this to the root element of the dataviews probably.

Reply:
I'm thinking we should only have in `dataviews-wrapper` no?

Reply:
I guess the page-header is outside of the dataviews component. So yeah, maybe two containers.

Reply:
One way to check this is to test the dataviews in storybook and see if the spacing is correct there.

Reply:
Oh good point. Yes we'll probably need two containers.